.github/ci/before_cmake.sh

```diff
@@ -7,12 +7,16 @@ BUILD_DIR=`pwd`
 cd /tmp
 
 # check that we can compile USD from sources (only Focal)
+return_code=0
+if [ "$(lsb_release -r -s)" != "20.04" ]; then
+  return_code=$(($return_code + 1))
```

Comment:
since `return_code` is initialized to `0` above, can this be simplified to `return_code=1`?

Reply:
I think it's easier to add new code and move things around in its current form.